src/follower.py

```diff
@@ -117,32 +128,47 @@ def reset_camera(self):
     will return False.
     """
     def detect(self):
-        obj_data = self._tag.detect()
-        detected = False
-
-        if obj_data:
-            detected = True
-            self._distance = obj_data['z']
-            self._turn_angle = np.degrees(obj_data['direction'])
-            self._decision = obj_data['decision']
+        tag_data = self._tag.detect()
 
-        return detected
+        if tag_data:
+            self._distance = tag_data['z']
+            self._turn_angle = np.degrees(tag_data['direction'])
+            self._decision = tag_data['decision']
+            return tag_data
+        else:
+            return False
 
+    """
+    Keeps the AR tag in view of the camera to avoid losing tag.
+    """
 
+    def track_tag(self):
+        tag_data = self.detect()
+        pan_angle = 5
+        
+        if tag_data == False:
+            self.stop()
```

Comment:
This was already happening. This script isn't about finding the tag after we've lost it. It's about never losing the tag in the first place. Get rid of this part.

Reply:
got rid of it